index.js

```diff
@@ -70,6 +78,83 @@ function MediaSession(opts) {
 }
 
 
+function offerAnswerQueue(self, errorMsg, jingleDesc, cb) {
+  self.q.push(function(qCb) {
+    function done(err, answer) {
+      qCb(err);
+      return cb(err, answer);
```

Comment:
Its probably worth this passing only one of the args through for safety. Something like `return (err ? cb(err) : cb(null, answer))`